packages/react-scripts/template/README.md

```diff
@@ -2082,6 +2090,46 @@ npm run build
 npm run analyze
 ```
 
+### Using Webpack Stats JSON
+
+> Note: this feature is available with react-scripts@2.0 and higher.
+
+Webpack can produce a JSON manifest that details the bundles, and several tools can use that file to do analysis.
+
+Unlike with sourcemaps, the JSON file isn't created automatically on build. You must pass a `--stats` flag:
+
+```sh
+npm run build -- --stats
+```
+
+Once the build is complete, you should have a JSON file located at `build/bundle-stats.json`.
+
+The quickest way to get insight into your bundle is to drag and drop that JSON file into [Webpack Visualizer](https://chrisbateman.github.io/webpack-visualizer/).
+
+Another very popular tool is [`webpack-bundle-analyzer`](https://www.npmjs.com/package/webpack-bundle-analyzer).
```

Comment:
Should we link to the GitHub page instead? https://github.com/webpack-contrib/webpack-bundle-analyzer

The README gets a bit squished on the https://www.npmjs.com/package/webpack-bundle-analyzer page 😕. In my opinion, the GitHub rendered page looks nicer.

In any case, this is only a nitpick and can be kept as-is :relaxed: